ignite/cmd/network_chain_init.go

```diff
@@ -100,7 +97,7 @@ func networkChainInitHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	c, err := nb.Chain(networkchain.SourceLaunch(chainLaunch))
+	c, err := nb.Chain(networkchain.SourceLaunch(chainLaunch), networkchain.CollectEvents(session.EventBus()))
```

Comment:
Already passed during newNetworkBuilder. We should somehow prevent passing it to Network and Chain by mistake.

Reply:
I forgot to remove this options when was refactoring `NetworkBuilder` constructor. I think it will not be the case in the future